active/progress/ProgressRing.md

```diff
@@ -0,0 +1,157 @@
+ Represents a control that indicates that an operation is ongoing. The typical visual appearance is a ring-shaped "spinner" that animates a filled area as progress continues.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+[Please see this table of current ProgressBar and ProgressRing modes and states.](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/kathyang/progress-styles/active/progress-styles/progress-styles.md)
+
+The following examples show how to use the IsIndeterminate property to change the mode of the ProgressRing and the Value property to change the proportionate amount indicated. 
+
+## Indeterminate ProgressRing
+
+```xml
+<ProgressRing IsActive="True" Height="100" Width="100" IsIndeterminate="True"/>
+```
+
+![](images/ProgressRing-indeterminate.PNG)
+## Determinate ProgressRing
+
+```xml
+<ProgressRing IsActive="True" Height="100" Width="100" Value="75"/>
+```
+![](images/ProgressRing-determinate.PNG)
+
+Below are tables showing how ProgressRing is affected by the IsDeterminate, IsActive, Value, ShowPaused, and ShowError properties.
+
+## Not Active ProgressRing
+| Note | Image |
+|:--|:-:|
+| When IsActive is False, ProgressRing will always appear blank, regardless of other properties | ![](images/ProgressRing-determinate-not-active.PNG) |
+
+## Indeterminate ProgressRing
+IsDeterminate is True
+
+| Value | ShowPaused | ShowError| Image | 
+|:--|:-:| :-:| :-:|
+|  | False | False | ![](images/ProgressRing-indeterminate.PNG) |
+|  | True | False | ![](images/ProgressRing-determinate-empty.PNG) |
+|  | False | True | ![](images/ProgressRing-determinate-empty.PNG) |
+|  | True | True | ![](images/ProgressRing-determinate-empty.PNG) | 
+| 75 | False | False | ![](images/ProgressRing-indeterminate.PNG) |
+| 75 | True | False | ![](images/ProgressRing-determinate-empty.PNG) |
+| 75 | False | True | ![](images/ProgressRing-determinate-empty.PNG) |
+| 75 | True | True | ![](images/ProgressRing-determinate-empty.PNG) | 
+
+
+## Determinate ProgressRing
+IsDeterminate is False
+
+| Value | ShowPaused | ShowError| Image | 
+|:--|:-:| :-:| :-:|
+|  | False | False | ![](images/ProgressRing-determinate-empty.PNG) |
+|  | True | False | ![](images/ProgressRing-determinate-empty.PNG) |
+|  | False | True | ![](images/ProgressRing-determinate-empty.PNG) |
+|  | True | True | ![](images/ProgressRing-determinate-empty.PNG) | 
+| 0 | False | False | ![](images/ProgressRing-determinate-empty.PNG) |
+| 75 | False | False | ![](images/ProgressRing-determinate.PNG) |
```

Comment:
Should this be 0.75 to align with [ProgressBar's default range of 0-1](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.progressbar)?

It might be useful to add Min and Max values to the table as well (particularly if this will inherit from RangeBase).

Reply:
Actually after testing a solo ProgressBar, the default range is 0-100 (but it inherits properties from RangeBase to modify the min and max values) .

I've added Min and Max to the API table and limited examples (since designs are all preliminary).